### ISSUE TITLE
[Broken Website] Gmail Reply / More Buttons

### ISSUE THREAD

**Issue by tophattwaffle** [darkreader/darkreader#9419]
# Broken Website Report

## Website Address
mail.google.com

## Steps To Reproduce
1. Use any Gmail Theme
2. Open any message from your inbox / all mail

## Problematic Observation

The reply and more buttons in the top right are the same color as the background 

## Screenshots

![image](https://user-images.githubusercontent.com/6774125/181104319-f58984e7-10df-4e2c-be45-49637bd40314.png)

## System Information

- OS: Windows 
- Browser: Chrome 103.0.5060.134
- Dark Reader Version: 4.9.52

**Comment by RobbyCBennett** [comment]
Also, most of the sidebar icons are missing in dark mode.

![light](https://user-images.githubusercontent.com/22121365/181396867-8278860a-f353-4bcc-9428-3ec37c9fd8a7.png)
![dark](https://user-images.githubusercontent.com/22121365/181396873-6675a814-04f2-4888-86fe-47d7406f1f84.png)

**Comment by RobbyCBennett** [comment]
I also discovered the same issue with the reply and forward arrows inside of the 3 dot menu.

Here's a quick fix for all of the issues.
```css
.qj:not(.aEe), .G-asx, .CL::before, .T-I-J3, .J-N-JX {
    filter: brightness(10);
}
```

**Comment by Gusted** [comment]
The inversion of images/icons are deferred for performance reasons. It can take on some machine a while before they are actually modified. Could you give ~5 minutes and see if the issue is still there?

**Comment by RobbyCBennett** [comment]
The problems remain after waiting on the website for 5 minutes. There are no changes on my machine after the page appears to have finished rendering.

**Comment by tophattwaffle** [comment]
Same for me. Left the page open for a while and no change.

**Comment by RobbyCBennett** [comment]
Also, another one I noticed:

![bad](https://user-images.githubusercontent.com/22121365/181668140-514f5b06-104d-4ecc-8edb-9367a2309cac.png)

![fixed](https://user-images.githubusercontent.com/22121365/181668150-20c584e3-91f0-465f-b5b0-c1bdb1809a9f.png)

```css
/* Removing this inversion fixes the problem */
.aiC
```

**Comment by Gusted** [comment]
Not able to reproduce the icons one.
![image](https://user-images.githubusercontent.com/25481501/181823085-a2440562-aec6-40b6-8be7-e7397284529e.png)

@RobbyCBennett Yes that seems to be broken(but seperate issue, It's removed by #9436)

**Comment by Gusted** [comment]
@RobbyCBennett Could you check if there's a selector that is inverting the icons(thus double inverting and therefor broken)

**Comment by tophattwaffle** [comment]
@Gusted 

That looks like it is the old gmail interface. Where as I'm on the new one. Note the "Compose" button of the left for an easy indicator of what UI you're on.

For some reason I can see the icons a bit now. Dark reader is the same version as when I opened this issue. Chrome is still Version 103.0.5060.134.

![image](https://user-images.githubusercontent.com/6774125/181824039-d722d152-fdc3-419c-a7ac-bd5a0e1ec6a1.png)

**Comment by Gusted** [comment]
Ah, A/B testing like usual. I'm quite unlucky, I still will have that old design for a while I guess.

**Comment by zeroepoch** [comment]
I switched back to the old interface for now because otherwise I can't see the folder icons. It's sad that Gmail still doesn't have a proper desktop dark mode.

**Comment by mjknoll** [comment]
Just happened to me after a refresh, fix posted above works.

**Comment by alexanderby** [comment]
I've got the icons problem with one of my Gmail accounts today. The fix was simple, but pushing updates on Friday is a bad idea, because of unpredictable reviews duration. And also we have many untested changes.

So, currently I suggest to use Filter mode for it, hope the fix will arrive to you next week.

**Comment by UweKeim** [comment]
Since no one mentioned the HTML editor toolbar when writing a new mail, I'll post my screenshot here, too:

![image](https://user-images.githubusercontent.com/415439/183506369-8d509d9d-d1dc-4166-b585-ce835a51b155.png)

(Although I do believe this is also fixed with the upcoming fix)

**Comment by UweKeim** [comment]
For me, the "Close" and "Minimize" buttons when composing a new message are still "broken":

![image](https://user-images.githubusercontent.com/415439/188935124-0fef475b-ce5d-4050-a758-430590ca8707.png)

(Extension version 4.9.57)

**Comment by RobbyCBennett** [comment]
Add the classes `.Hl, .Ha`

```css
.qj:not(.aEe), .G-asx, .CL::before, .T-I-J3, .J-N-JX, .Hl, .Ha {
    filter: brightness(10);
}
```

**Comment by Gusted** [comment]
> Therefore I would prefer that this will find its way into the extension itself some time in the future to be more helpful.

https://github.com/darkreader/darkreader/pull/9795#issue-1368586405 Will do the job. Feel free to check out [the README](https://github.com/darkreader/darkreader#site-fixes) to pull in the sites fixes, before the next release.

**Comment by dracan** [comment]
> > Therefore I would prefer that this will find its way into the extension itself some time in the future to be more helpful.
> 
> [#9795 (comment)](https://github.com/darkreader/darkreader/pull/9795#issue-1368586405) Will do the job. Feel free to check out [the README](https://github.com/darkreader/darkreader#site-fixes) to pull in the sites fixes, before the next release.

I'm having no luck with this. I get lots of gmail icons being black (so pretty much invisible). I've followed your instructions to sync from Github (including the reset), but doesn't seem to make any difference...

![image](https://user-images.githubusercontent.com/567988/189480557-3d5217bb-d608-41bd-b5a9-205be2bd23ad.png)

![image](https://user-images.githubusercontent.com/567988/189480562-895d22da-b8a5-4bd3-8142-45fb01d6c05e.png)

**Comment by Gusted** [comment]
That seems to be another problem(seem you're still on the old UI), did this only happen recently? Feel free to open up a new issue for this.

**Comment by dracan** [comment]
> That seems to be another problem(seem you're still on the old UI), did this only happen recently? Feel free to open up a new issue for this.

Thanks - new ticket raised here: #9796 🙂
